packages/flame/lib/src/particles/accelerated_particle.dart

```diff
@@ -1,36 +1,45 @@
 import 'dart:ui';
 
+import '../../extensions.dart';
 import '../components/mixins/single_child_particle.dart';
 import 'curved_particle.dart';
 import 'particle.dart';
 
-/// A particle serves as a container for basic
-/// acceleration physics.
-/// [Offset] unit is logical px per second.
-/// speed = Offset(0, 100) is 100 logical pixels per second, down
-/// acceleration = Offset(-40, 0) will accelerate to left at rate of 40 px/s
+/// A particle that serves as a container for basic acceleration physics.
+///
+/// [Vector2] unit is logical px per second.
+///
+/// ```dart
+/// AcceleratedParticle(
+///   speed: Vector2(0, 100), // is 100 logical pixels per second down.
+///   acceleration: Vector2(-40, 0) will accelerate to the left at rate of 40 px/s
+/// )
+/// ```
 class AcceleratedParticle extends CurvedParticle with SingleChildParticle {
   @override
   Particle child;
 
-  final Offset acceleration;
-  Offset speed;
-  Offset position;
+  final Vector2 acceleration;
```

Comment:
Don't know if we have code style for that, but I would remove these blank lines

Reply:
Hmm my preference is a blank line between them. Because you also do that when adding dartdocs.

I am btw planning on documenting these particles a bit more in a follow up PR.

Reply:
I agree with Erick, but if there are dartdocs it should be a line inbetween.